packages/react/src/components/form/Search/SearchInput.tsx

```diff
@@ -0,0 +1,26 @@
+import { forwardRef } from 'react';
+import { Input } from '../Input';
+
+export type SearchInputProps = Omit<
+  React.InputHTMLAttributes<HTMLInputElement>,
+  'readOnly' | 'type'
+> &
+  (
+    | { 'aria-label': string; 'aria-labelledby'?: never }
+    | { 'aria-label'?: never; 'aria-labelledby': string }
+  );
+
+export const SearchInput = forwardRef<HTMLInputElement, SearchInputProps>(
+  ({ className, ...rest }, ref) => {
+    return (
+      <Input
+        ref={ref}
+        {...rest}
+        type='search'
+        /* We need an empty placeholder for the clear button to be able to show/hide */
+        placeholder=''
+        {...rest}
```

Comment:
Double rest spread here, I assume we want the first as this is an input used for a specific use-case?

Reply:
Nice catch, I kept the last one. Thought it would be weird, I think we should keep with our pattern and let consumers override